quizzes/ch11-03-smart_pointers.toml

```
@@ -34,6 +34,13 @@ fn main() {
 """
 answer.doesCompile = true
 answer.stdout = "Peter"
+context = """
+The `student1` variable was first instantiated as a smart pointer to an instance of the struct `Student`.
+When we assigned `student1` to a new variable `student2`, we merely copied the _smart pointer_ to the previously created struct,
+so both variables pointed to the same struct in memory.
```

```suggestion
so both variables referred to the same struct in memory.
```

I like to keep `point` for pointer-related terms